build.gradle

```diff
@@ -2,19 +2,33 @@ buildscript {
     repositories {
         mavenLocal()
         mavenCentral()
+        jcenter()
+        google()
     }
     dependencies {
         classpath 'com.mparticle:android-kit-plugin:+'
     }
 }
 
+allprojects {
```

Comment:
please remove this entire block.

Reply:
The example is a little out of date, here's an example kit build.gradle from another project:

```
buildscript {
    repositories {
        google()
        mavenLocal()
        jcenter()
    }
    dependencies {
        classpath 'com.mparticle:android-kit-plugin:' + project.version
    }
}

apply plugin: 'com.mparticle.kit'

android {
    defaultConfig {
        minSdkVersion 14
    }
}

dependencies {
    api 'com.appsee:appsee-android:2.4.4'
}
```